superset/tasks/schedules.py

```diff
@@ -637,6 +658,8 @@ def deliver_slack_alert(alert_content: AlertContent, slack_channel: str) -> None
             "slack/alert_no_screenshot.txt",
             label=alert_content.label,
             sql=alert_content.sql,
+            observation_value=alert_content.observation_value,
```

Comment:
when possible it would be nice to have a custom error message e.g.
```
SQL Result: 2561720.0
Expected: 2561720.0 < 100000
```

Reply:
I can add that in another PR

Reply:
sgtm - leave a TODO